src/models/Sheets.js

```diff
@@ -1,17 +1,37 @@
-const data = require('../mockdata');
 const Sheet = require('./Sheet');
+const fetch = require('isomorphic-fetch');
 /* eslint-disable class-methods-use-this */
+function getJson(response) {
+  return response.json();
+}
+function createSheetInstances(sheets) {
+  return sheets.map(sheet => new Sheet(sheet));
+}
 class Sheets {
   constructor({ spreadsheetId, sheetName }) {
     this.spreadsheetId = spreadsheetId;
     this.sheetName = sheetName;
+    this.sheetUrl = `https://sheets.googleapis.com/v4/spreadsheets/${this.spreadsheetId}/values/${sheetName}!A1:ZZ?access_token=${process.env.TKN}`;
+    this.spreadSheetUrl = `https://sheets.googleapis.com/v4/spreadsheets/${this.spreadsheetId}?access_token=${process.env.TKN}`;
   }
   get sheets() {
-    return this.sheetName ? [new Sheet(data.SHEET1_DATA)] : this.getAllUserSheets();
+    return this.sheetName ? this.getSheetByName() : this.getAllUserSheets();
+  }
+  getSheetByName() {
+    return fetch(this.sheetUrl)
+      .then(response => getJson(response))
+      .then(data => [new Sheet(data)]);
   }
-  getAllUserSheets() {
-    const sheetDataArr = [data.SHEET1_DATA, data.SHEET2_DATA];
-    return sheetDataArr.map(sheetData => new Sheet(sheetData));
+  async getAllUserSheets() {
```

Comment:
@michaelmurray6298 I think you should remove `async` here since we are not using `await` anywhere in the function definition.